src/js/tbody.js

```diff
@@ -52,7 +52,7 @@ export class RegularBodyViewModel extends ViewModel {
             metadata.value = null;
             metadata.row_path = null;
         } else if (formatter) {
-            formatter.format(td, val, type, val.length === depth, is_open);
+            formatter.format(td, val, type, val.length >= depth, is_open);
```

Comment:
Breaks perspective integration, which is not currently enabled in the test suite.  Formatting is another feature destined for the trash heap regardless, as this is better handled within the data model.

Reply:
Can you please help me understand the issue here a little better? The `perspective.html` example in the latest version of this PR does seem to correctly render the tree in its first column, so it would be helpful to know exactly what this line breaks.